<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserForm</class>
 <widget class="QWidget" name="UserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>300</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Управление USB ключем.</string>
  </property>
  <widget class="QLineEdit" name="lineEditProductKey">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>WWWWW-WWWWW-WWWWW-WWWWW-WWWWW</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelKeyInfo">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ключ продукта.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSendProductKey">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить ключ продукта.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSetRSA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Установить ключ шифрования канального уровня.</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>485</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
